src/client/README.md

```diff
@@ -0,0 +1,84 @@
+# Client
+
+This project was generated using [Nx](https://nx.dev).
+
+<p align="center"><img src="https://raw.githubusercontent.com/nrwl/nx/master/nx-logo.png" width="450"></p>
+
+🔎 **Nx is a set of Angular CLI power-ups for modern development.**
+
+## Quick Start & Documentation
+
+[Nx Documentation](https://nx.dev)
+
+[30-minute video showing all Nx features](https://nx.dev/getting-started/what-is-nx)
+
+[Interactive Tutorial](https://nx.dev/tutorial/01-create-application)
+
+## Adding capabilities to your workspace
+
+Nx supports many plugins which add capabilities for developing different types of applications and different tools.
+
+These capabilities include generating applications, libraries, .etc as well as the devtools to test, and build projects as well.
+
+Below are some plugins which you can add to your workspace:
+
+- [Angular](https://angular.io)
+  - `ng add @nrwl/angular`
+- [React](https://reactjs.org)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-WebForms/pullRequest?prid=5102156)